lib/mix/lib/mix/tasks/archive.uninstall.ex

```diff
@@ -8,8 +8,18 @@ defmodule Mix.Tasks.Archive.Uninstall do
 
       mix archive.uninstall archive.ez
 
+  ## Command line options
+
+    * `--sha512` - checks the archive matches the given SHA-512 checksum. Only
+      applies to installations via URL or local path
```

Comment:
This option does not apply here, does it?

Reply:
Bad copy/paste. Thanks!